apps/list.c

```diff
@@ -70,6 +72,10 @@ static void list_ciphers(void)
     BIO_printf(bio_out, "Legacy:\n");
     EVP_CIPHER_do_all_sorted(legacy_cipher_fn, bio_out);
 
+    if (ciphers == NULL) {
+        BIO_printf(bio_err, "ERROR: Memory allocation\n");
+        return;
+    }
```

Comment:
The seems to be a repetition of the code just a few lines above?

Reply:
A merge addition.  Will remove.